source/documentation/deploying_apps/deploying_java.md

```diff
@@ -0,0 +1,122 @@
+## Deploy a Java app
+
+
+This section covers how to deploy a Java application to GOV.UK PaaS using the [Java buildpack](https://github.com/cloudfoundry/java-buildpack) [external link]. 
+
+
+### Deploying a JAR file
+
+If your java application can be packaged as a [self-executable JAR](https://github.com/cloudfoundry/java-buildpack/blob/master/docs/container-java_main.md) [external link] then deployment can be as simple as using `cf push -p [your-app].jar [your-app-name]`.
+
+This example will walk through creating a simple "Hello World" application that embeds the popular ([Jetty](http://www.eclipse.org/jetty/)) [external link] webserver.
+The instructions assume you have already carried out the setup process explained in the [Quick Setup Guide](https://docs.cloud.service.gov.uk/#quick-setup-guide) section.
+
+1. Create a directory for your Java application:
+
+    ```bash
+    mkdir myapp
+    cd myapp
+    ```
+
+1. Download Jetty.
+
+    ```bash
+    curl -o ./jetty.jar http://central.maven.org/maven2/org/eclipse/jetty/aggregate/jetty-all/9.4.6.v20170531/jetty-all-9.4.6.v20170531-uber.jar
```

Comment:
Can we make this link to the latest jetty instead? Or failing that the latest 9.x? 

I know this is only a getting started guide, but it would be preferable if in a couple of months we aren't advising people to use an outdated version with security vulnerabilities.

Reply:
I've added in a call to action and link to the latest version page for people to check.